compiler/src/iree/compiler/DispatchCreation/test/bubble_up_expand_shapes.mlir

```diff
@@ -0,0 +1,24 @@
+// RUN: iree-opt --split-input-file --mlir-print-local-scope --pass-pipeline="builtin.module(util.func(iree-dispatch-creation-bubble-up-expand-shapes, canonicalize, cse, canonicalize))" %s | FileCheck %s
```

Comment:
Do we need to run `canonicalize, cse, canonicalize` in the test? Can we drop it?